src/uss/ZoweUSSNode.ts

```diff
@@ -580,6 +553,49 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
      *
      */
     public getUSSDocumentFilePath() {
-        return path.join(extension.USS_DIR || "", "/" + this.getSessionNode().getProfileName() + "/", this.fullPath);
+        return path.join(globals.USS_DIR || "", "/" + this.getSessionNode().getProfileName() + "/", this.fullPath);
+    }
+
+    private returnmProfileName(): string {
+        return this.mProfileName;
+    }
+}
+
+let wasSavedRecently = false;
+let saveListener = null;
```

Comment:
Should these be part of a separate class/helper file (along with the 3 functions below)?
Maybe that way it's easier to document and maintain?